core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java

```diff
@@ -17,6 +17,13 @@
 
   private SerializableStrategy strategy;
 
+  // for two-phase consensus commit
+  public static final String TWO_PHASE_CONSENSUS_COMMIT_PREFIX = PREFIX + "2pcc.";
+  public static final String ACTIVE_TRANSACTIONS_MANAGEMENT_ENABLED =
+      TWO_PHASE_CONSENSUS_COMMIT_PREFIX + "active_transactions_management_enabled";
```

Comment:
Sorry, I meant `.enabled` like this.
https://spark.apache.org/docs/latest/security.html#encryption

For this, I don't think it is very meaningful since no other options are available for `active_transactions_management` but I think this rule is useful when we have some other options like `active_transactions_management.xxx`.
What do you think?